pkgs/development/libraries/linbox/default.nix

```diff
@@ -63,14 +63,18 @@ stdenv.mkDerivation rec {
     })
   ];
 
-  doCheck = true;
+  enableParallelBuilding = true;
+
+  # needed for darwin
+  doInstallCheck = true;
+  installCheckTarget = "check";
```

Comment:
Same as #45001, please make the reason clear in the comment and set `doCheck = false`. Maybe you should report the issue upstream.

Reply:
You can then add a link to the upstream issue to the comment.

Reply:
See https://github.com/NixOS/nixpkgs/pull/44248, this is a pretty common pattern for packages that test their own libraries before they are installed.

Reply:
I personally think that this is a much better test, messing with LD_LIBRARY_PATH or rpaths like what happens on linux provides no guarantee that the libraries actually work properly once installed.

Reply:
But this doesn't either: It still checks the built binary, not the installed one.

If a package has proper install checks and we someday default `doInstallCheck = true`, they wouldn't be executed in this case. Also it requires a bunch of manual changes.

Reply:
In general I agree that install checks are more useful than build checks for packaging.

Reply:
No this is the opposite: it's testing against the installed binary even though the tests want to test against the built binary. Even though the Makefile explicitly passes flags to point the tests at the built binary, Nix must be overriding them to point it at the installed binary because in general it wants everything to point there. So I think `doInstallCheck` is fine.

Reply:
Did you test that by removing the build results before running the tests? I would be surprised if nix did anything behind the scenes to make it use the installed binary. I would think that the built binary just looks for the installed library.

Anyways, thanks for adding support! Although I don't entirely like the `InstallCheck` result, I think with the comment it is a worthy tradeoff for darwin support.